website/src/pages/index.js

```diff
@@ -50,7 +49,7 @@ class Index extends React.Component {
 					}
 					buttonText={<translate>Get Started</translate>}
 					buttonUrl={`tutorials`}
-				/>
+				/> */}
 
 				<section className='mainContainer' id='home'>
```

Comment:
this is not valid JSX? you should use double quotes, don't know why you have single quotes everywhere, didn't even know it was possible before :o

Reply:
oh, actually seems to work :o